.circleci/config.yml

```diff
@@ -0,0 +1,127 @@
+version: 2
+
+defaults: &defaults
+  working_directory: ~/circleci-champaign-ak-processor
+  docker:
+    - image: soutech/ruby2.4.1_node_awscli:0.2
+
+jobs:
+  install_dependencies:
+    <<: *defaults
+    steps:
+      - checkout
+      - restore_cache:
+          name: Restoring bundler cache
+          key: gem-cache-{{ checksum "Gemfile.lock" }}
+      - run:
+          name: Install gems
+          command: bundle install --jobs 4 --deployment
+      - save_cache:
+          key: gem-cache-{{ checksum "Gemfile.lock" }}
+          paths:
+          - vendor/bundle
+      - persist_to_workspace:
+          root: .
+          paths:
+            - vendor/bundle
+            - .bundle
+
+  rspec:
+    docker:
+      - image: soutech/ruby2.4.1_node_awscli:0.2
+        environment:
+          PG_USERNAME: champaign
```

Comment:
These I copied over from the env.yml file for the docker-compose setup we were using to run the tests before.